bindings/web5_uniffi/src/web5.udl

```diff
@@ -264,7 +264,7 @@ interface VerifiableCredential {
   constructor(
     string json_serialized_issuer,
     string json_serialized_credential_subject,
-    VerifiableCredentialCreateOptionsData? options
+    VerifiableCredentialCreateOptionsData options
```

Comment:
this should be optional, the APID is incorrect in its current state but the code was right. unless you think it should be required?